src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigsCollection.java

```diff
@@ -85,50 +49,52 @@ public String toPackedFasta(){
      * @param packedFastaLine
      */
     @VisibleForTesting
-    protected static ContigsCollection fromPackedFasta(final String packedFastaLine) {
+    protected static ContigsCollection fromPackedFasta(final long assemblyID, final String packedFastaLine) {
```

Comment:
I'm a little bit confused about `assemblyId`. Why do we need it to create a `ContigsCollection`? And then why store it at the level of the `LocalAssemblyContig`? It doesn't look like it's being accessed there. If we want to track it it seems like it would go better at the ContigsCollection level.